qemu/tests/virtio_fs_subtest_during_io.py

```diff
@@ -0,0 +1,109 @@
+import time
+import os
+
+from virttest import utils_test, utils_misc, utils_disk
+from virttest import error_context
+from virttest import data_dir
+from provider import win_driver_utils
+from provider import virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Stop/continue the vm during the virtiofs test.
+
+    1) Boot guest w/ virtiofs device.
+    2) For windows guest, enable driver verifier first.
+    3) Run virtiofs function test.
+    4) During steps 3, stop vm and then resume vm.
+    5) Memory leak check.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    @error_context.context_aware
+    def stop_resume_vm_during_io_test():
+        start_time = time.time()  # record the start time.
+        # run time expected, the unit is second.
+        max_run_time = params.get_numeric("max_run_time", 30)
+        while time.time() - start_time < max_run_time:
+            if basic_io_test.is_alive():
+                if os.path.exists(host_data) and os.path.getsize(host_data) > 0:
+                    error_context.context("The file has been detected: %s" % host_data,
```

Comment:
I think this is the log info not steps info, so using test.log.info( ) is better.